src/components/add-transaction-dialog/add-transaction-dialog.html

```diff
@@ -0,0 +1,116 @@
+<aside class="mdc-dialog add-transaction-dialog" role="alertdialog" aria-labelledby="my-mdc-dialog-label" aria-describedby="my-mdc-dialog-description">
+  <div class="mdc-dialog__surface">
+
+    <header class="mdc-dialog__header">
+      <h2 class="mdc-dialog__header__title">
+        Add transaction
+      </h2>
+    </header>
+
+    <section class="mdc-dialog__body">
+      <div class="mdc-layout-grid add-transaction-dialog__row">
+        <div class="mdc-layout-grid__inner">
+          <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-3">
+            <h3 class="mdc-typography--subheading2">Type of transaction: </h3>
+          </div>
+          <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-9">
+            <div class="mdc-form-field">
+              <div class="mdc-radio add-transaction-dialog__expence">
+                <input class="mdc-radio__native-control" type="radio" id="add-transaction-dialog-expense" name="radios" checked>
+                <div class="mdc-radio__background">
+                  <div class="mdc-radio__outer-circle"></div>
+                  <div class="mdc-radio__inner-circle"></div>
+                </div>
+              </div>
+              <label for="add-transaction-dialog-expense">Expence</label>
+            </div>
+            <div class="mdc-form-field">
+              <div class="mdc-radio add-transaction-dialog__income">
+                <input class="mdc-radio__native-control" type="radio" id="add-transaction-dialog-income" name="radios" checked>
+                <div class="mdc-radio__background">
```

Comment:
do you really need id here? maybe label could work without `for`? please check

Reply:
yes, I heart you yesterday and managed to make textfields work without ID.
and no, it can't. radio doesn't work correctly

Reply:
ok